burr/integrations/persisters/test_mongodb_persister.py

```diff
@@ -0,0 +1,29 @@
+import unittest
```

Comment:
Thanks for adding a test. I realize we haven't added ones for the others yet.

Could you move this under tests please, and create the right folder structure while you're at it.

You'll then need to add `mongodb` as a dependency under `tests` in pyproject.toml for the tests to execute in CI.

Reply:
I have created  folder named persisters inside tests/integrations/
and moved my test  file,  'test_mongodb_persister.py' inside  that  also added suggested changes.

Reply:
sorry i misspoke -- I think you need to add this package https://pypi.org/project/pymongo/ as mongodb isn't a python package name.